includes/compat/class-wc-stripe-email-customer-failed-refund.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Failed Refund Notification (shopper)
+ *
+ * @since 9.6.0
+ */
+class WC_Stripe_Email_Customer_Failed_Refund extends WC_Stripe_Email_Failed_Refund {
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		parent::__construct();
+
+		$this->id          = 'wc_stripe_failed_refund_customer';
+		$this->description = __( 'Sent to a customer when a refund request fails. The email contains the original order information.', 'woocommerce-gateway-stripe' );
```

Comment:
To match the details, I think it's worth simplifying and removing the "request". I also think we should call out that it occurs for cancellations, so merchants know that will happen.
```suggestion
		$this->description = __( 'Sent to a customer when a refund fails or is cancelled. The email contains the original order information.', 'woocommerce-gateway-stripe' );
```